### ISSUE TITLE
Arrived aircraft will show in the list

### ISSUE THREAD

**Issue by neilenns** [neilenns/vatsim-plan-verifier#490]
If a flight departs a watched airport (e.g. KPDX) and flies then lands at the destination the flight plan will show again in the vatsim list because they are not moving.

Sample record of a flight that left KPDX and landed but showed up again:

```javascript
{
  "_id": {
    "$oid": "64d13f89661a3de5f12b8b1f"
  },
  "callsign": "DAL3133",
  "groundspeed": 0,
  "rawAircraftType": "H/B78X/L",
  "departure": "KPDX",
  "arrival": "PANC",
  "cruiseAltitude": 380,
  "route": "ARRIE TOU PRYCE BKA LAIRE JOH WITTI4",
  "squawk": "1545",
  "remarks": "PBN/A1B1C1D1L1O1S2 NAV/RNVD1E2A1 DOF/230807 REG/N891SB EET/CZVR0034 PAZA0139 RMK/THE DELTA VIRTUAL /V/",
  "__v": 0
}
```

**Comment by neilenns** [comment]
This will require #492 before anything can be done about it

**Comment by neilenns** [comment]
This was fixed with #492
